libs/watt/package/menu/index.ts

```diff
@@ -16,7 +16,22 @@
  * limitations under the License.
  */
 //#endregion
+// Individual exports (kept for backward compatibility)
```

Comment:
```suggestion
// Individual exports
```

Not really true IMO, you can still require the individual exports (fx. for child queries)